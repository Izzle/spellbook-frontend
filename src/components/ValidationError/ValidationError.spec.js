import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import ValidationError from './ValidationError';


describe('<ValidationError />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<ValidationError message={'Test ERROR'}/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as expected', () => {
        const tree = renderer.create(<ValidationError message={'Test ERROR'}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });    
});