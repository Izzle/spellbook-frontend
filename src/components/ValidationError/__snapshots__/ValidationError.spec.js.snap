// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ValidationError /> renders this UI as expected 1`] = `
<div
  className="validation_error"
>
  Test ERROR
</div>
`;
